src/main/java/seedu/bigpp/command/viewercommand/ViewerAddCommand.java

```diff
@@ -1,12 +1,20 @@
 package seedu.bigpp.command.viewercommand;
 
 import seedu.bigpp.command.Command;
+import seedu.bigpp.pc.PC;
+import seedu.bigpp.pc.PCList;
 
 public class ViewerAddCommand extends Command {
 
+    public ViewerAddCommand(String arguments) {
+        super.setArguments(arguments);
+    }
+
     @Override
     public String executeCommand() {
-        // TODO Auto-generated method stub
-        return "";
+        String name = super.getArguments();
+        PC pc = new PC(name, false);
+        PCList.addPC(pc);
+        return "I have created the PC with the following name: " + name;
```

Comment:
Maybe can use statement such as "Custom PC: [ name ] has been created" to follow format of pc viewer

Reply:
agreed, this change sounds good